.travis.yml

```diff
@@ -39,7 +37,7 @@ before_install:
 
 before_script:
   # Installs TYPO3
-  - composer require typo3/cms=$TYPO3_VERSION
+  - composer require typo3/cms-core=$TYPO3_VERSION
```

Comment:
It would be cool if you can also create a new TYPO3V9 branch for https://github.com/AOEpeople/TYPO3_RestlerExamples
So we can check if all functionality which works on TYPO3V7 will also work in TYPO3V9.
There are behat tests belong to the unittests.

Reply:
made a issue, will look into this in a few days